### PR TITLE
Migrate integration test qemu version to 7.0.0

### PR DESCRIPTION
**Pull request by 10000TB**
(bump up major version to reflect major upgrade to qemu)

### CODE REVIEW

**Comment by 10000TB** [comment]
docker builds ok. I cant push atm.

```
docker push uroottest/test-image-arm:5.0.0
The push refers to repository [docker.io/uroottest/test-image-arm]
1d069c4bbfd3: Preparing
304089c242c1: Preparing
382fc00b9c9b: Preparing
26f9aab21dff: Preparing
0dcfaf3c97d2: Preparing
ccfb2532594a: Waiting
2ad7f9c1dfbf: Waiting
d97149b9174c: Waiting
67073526cab9: Waiting
6a889e8bc32b: Waiting
94734f6bed59: Waiting
994ac7d3da76: Waiting
d123bcfc2540: Waiting
dceb40a430c7: Waiting
1cdb5d93f7a8: Waiting
24e043e4c1be: Waiting
c8768156232f: Waiting
d4c2a7004ea2: Waiting
3b441d7cb46b: Waiting
d3710de04cb3: Waiting
91f7336bbfff: Waiting
e2e8c39e0f77: Waiting
denied: requested access to the resource is denied
```

**Review by rminnich — APPROVED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -146,7 +146,7 @@ jobs:
   test-integration-arm:
     <<: *integration-template
     docker:
-      - image: uroottest/test-image-arm:v4.4.0
+      - image: uroottest/test-image-arm:v5.0.0
```
Comment (rminnich) [inline .circleci/config.yml:149]: did you create the new docker image with this version? It seems not?

**Review by rminnich — CHANGES_REQUESTED** [review]

File: .circleci/config.yml (hunk 1 above)
Comment (10000TB) [reply .circleci/config.yml:149]: you are right, just sorted out some permission issue w/ pushing to docker hub. Just pushed, triggering a new build.

**Comment by 10000TB** [comment]
Test 1: {to be updated}

**Comment by 10000TB** [comment]
I haven't see arm board failing after the upgrade to qemu 7.  Maybe that is a good indication.

Now this PR upgraded all boards to qemu 7. 

PTAL. meanwhile, doing a few more triggers

**Review by rminnich — APPROVED** [review]

**Pull request merged**